dali/base/dafdesc.cpp

```diff
@@ -937,6 +941,7 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
     SocketEndpointArray *pending;   // for constructing cluster group
     bool setupdone;
     byte version;
+    unsigned short extendedVersion = 0;
```

Comment:
Normally you'd use the serialization version to validate during deserialization only, you wouldn't need or store it beyond that, so I'm not sure it needs to be a class member like this.